### DISCUSSION TITLE
Getting Chapter-Figure numbering in TOC

### DISCUSSION THREAD

**Discussion by 6Nokturnal6** [joaomlourenco/novathesis#434] [Q&A] [answered]
I just want to contribute by saying I 'm only able to get chapter-Figure numbering in TOC for EE-UMINHO when modifying the file `novathesis-7.0.0/NOVAthesisFiles/Schools/uminho/uminho-defaults.ldf` as follows:

```
\NTAddToHook{mainmatter/post}{%
  %\counterwithout{figure}{chapter}% 
  %\counterwithout{table}{chapter}%

  \numberwithin{figure}{chapter}%
  \numberwithin{table}{chapter}%
} 
```

edit: I'm running the template on Kile with texlive under GNU/Linux.

**Comment by joaomlourenco** [comment, marked as answer]
Are you sure you want to do that?
I believe the rules for UMINHO state that the numbering for figures and tables should be continuous and not “chapter.number”.

**Reply by joaomlourenco** [reply]
But if you really want to do it (at your own risk), just comment everything at it should work:

```latex
%\NTAddToHook{mainmatter/post}{%
%  \counterwithout{figure}{chapter}% 
%  \counterwithout{table}{chapter}%
%} 
```

**Reply by 6Nokturnal6** [reply]
> Are you sure you want to do that? I believe the rules for UMINHO state that the numbering for figures and tables should be continuous and not “chapter.number”.

Well, indeed! "As tabelas, quadros, gráficos, figuras, etc. devem ser numerados, em cada caso, de 1 a N, e conter um titulo, sintético, que traduza claramente o respetivo conteúdo." My superviser asked me to use Chapter-Figure. I'm correcting this and informing him. I did not take enough attention. Thanks.
